vhdbuilder/packer/packer-plugin.pkr.hcl

```diff
@@ -0,0 +1,8 @@
+packer {
+  required_plugins {
+    azure = {
+      version = "= 1.4.3"
```

Comment:
It seems like that packer version installed by default is `Packer v1.8.1`. Should we need this file to add v1.4.3 as the required?

Reply:
I used to use `>=1.4.3` but you asked me to use a fixed static version `1.4.3` in case of failure. This is the version that guarantees to work.

Reply:
btw this is the packer plugin version not the packer version.